creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -65,6 +66,23 @@ const MetricLabels = Object.freeze({
       'primary_and_or_secondaries', // A secondary gets promoted to new primary and one or both secondaries get replaced with new random nodes,
       'null' // No change was made to the user's replica set because the job short-circuited before selecting or was unable to select new node(s)
     ]
+  },
+  [MetricNames.FIND_SYNCS_RESULTS_TOTAL_GAUGE]: {
+    // The primary node being synced from. Dynamic values because it can be any node
+    primary: [],
+    // The secondary node being synced to. Dynamic values because it can be any node
+    secondary: [],
```

Comment:
i don't think we should be recording these since the cardinality is so high
just result is fine
with primary and secondary, each gauge would generate (numPrimaries * numSecondaries * num resultTypes) time series, or ~ 36000 on prod

Reply:
i think the rule of thumb is that any individual server shouldn't expose more than ~10000 time series ([per readme](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/prometheusMonitoring/README.md#labeling-and-cardinality))

there are a few diff attack angles here - e.g. we should add nodes where each node records outcome for every incoming sync request, which would answer that question

its worth thinking through a bit more how to holistically get insight without overloading prometheus

Reply:
ok gotcha, I just removed the primary/secondary from metrics and logged it instead. that's a solid idea on recording for incoming syncs. once we do that we should have everything covered for syncs -- finding, issuing, and receiving/executing

> > @SidSethi thanks for the review! I responded to the cardinality issue -- thinking if we should try to at least expose some info about which nodes are causing the metrics. maybe logging when a metric is recorded would be a good middle ground to let us dive into why/where certain values happen without overloading Prometheus? lmk what you think and I'll make some changes 🙂
> 
> agreed - i think as best practice, every time a metric is recorded there should be a corresponding log, so that seems like the right place to log the primary-secondary info. wdyt?

makes sense to me -- I went ahead and made that change and updated the PR description

Reply:
sweet